--- supertuxkart-0.9.3/lib/angelscript/source/as_config.h.0001~	2018-03-10 20:38:08.071130470 +0100
+++ supertuxkart-0.9.3/lib/angelscript/source/as_config.h	2018-03-10 20:41:25.800309290 +0100
@@ -854,6 +854,15 @@
 			// STDCALL is not available on 64bit Linux
 			#undef STDCALL
 			#define STDCALL
+		#elif defined(__aarch64__)
+			// AngelScript currently doesn't support native calling
+			// for 64bit ARM processors so it's necessary to turn on
+			// portability mode
+			#define AS_MAX_PORTABILITY
+
+			// STDCALL is not available on ARM
+			#undef STDCALL
+			#define STDCALL
 		#elif (defined(__ARMEL__) || defined(__arm__)) && !(defined(__ARM_ARCH_4__) || defined(__ARM_ARCH_4T__))
 			#define AS_ARM
 
